src/command.js

```diff
@@ -2,8 +2,8 @@ import isWindows from 'is-windows'
 
 export default commandConvert
 
-const envUseUnixRegex = /\$(\w+)/ // $my_var
-const envUseWinRegex = /%(.*?)%/ // %my_var%
+const envUseUnixRegex = /\$(\w+)|\${(\w+)}/g // $my_var or ${my_var}
+const envUseWinRegex = /%(.*?)%/g // %my_var%
```

Comment:
Do we need to be concerned about the `g` here based on your last PR? Perhaps we could just inline these regex declarations within the `commandConvert` function. I'm sure there's no relevant perf issues with that, and we could avoid the stateful nature of `g`.